src/ray/common/cgroup/cgroup_setup.cc

```diff
@@ -73,14 +80,55 @@ bool MoveProcsInSystemCgroup() {
   return out_file.good();
 }
 
+// This function is the reverse operation for [MoveProcsInSystemCgroup], which moves all
+// processes from raylet system cgroup into root cgroup.
+//
+// Return whether the PIDs move successfully.
+bool MoveProcsFromSystemCgroup() {
```

Comment:
this function looks overfit to the specific functionality. prefer something like:
`MoveProcesses(const std::string dest)`

where `dest` is either a string identifier or maybe even an enum that enumerates the set of cgroups that we care to manage

Reply:
> this function looks overfit to the specific functionality.

That's why it's placed in anonymous namespace in impl file, rather than a public header file.